addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -193,11 +194,22 @@ export class WebsocketWorker {
      * given client.
      * @param {Number} [param0.lastNotificationId] Last notification id
      * known by the client.
+     * @param {Number|undefined} [param0.uid] Current user id.
      */
-    _initializeConnection(client, { debug, lastNotificationId }) {
+    _initializeConnection(client, { debug, lastNotificationId, uid }) {
         this.lastNotificationId = lastNotificationId;
         this.debugModeByClient[client] = debug;
         this.isDebug = Object.values(this.debugModeByClient).some(debugValue => debugValue !== '');
+        if (!this._isWebsocketConnected()) {
+            return;
+        }
```

Comment:
Who is this related to change with user?

Reply:
I slightly changed the syntax but the point is not closing a socket which is not yet connected (it would lead to warnings).